### PR TITLE
 rework export setting blade JIRA:Vp223

### PR DESCRIPTION
**Pull request by AliveMen**
update UI for export-settings blade
receive data total count
save selected provider to local storage

Merge branch 'export/dev' into vp223_reeork_export_setting_blade

### CODE REVIEW

File: Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-settings.js [hunk 1]
```diff
@@ -125,6 +175,11 @@ angular.module('virtoCommerce.exportModule')
                 exportDataRequest: angular.copy(blade.exportDataRequest),
                 onCompleted: function(dataQuery) {
                     blade.exportDataRequest.dataQuery = dataQuery;
+                    blade.dataSelected =
+                        (blade.exportDataRequest.dataQuery.objectIds &&
+                            blade.exportDataRequest.dataQuery.objectIds.length > 0)
```
Comment (yecli) [inline Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-settings.js]: length > 0
No need in >0

File: Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-type-selector.js [hunk 2]
```diff
@@ -35,6 +35,7 @@ angular.module('virtoCommerce.exportModule')
                     var exportDataRequest = {
                         exportTypeName: selectedType.typeName,
                         allColumnsOfType: allColumnsOfType,
+                        isTabularExportSupported: selectedType.isTabularExportSupported,
```
Comment (yecli) [inline Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-type-selector.js]: Let's move `isTabularExportSupported` property to independent parameter to callback, to not pass it to server with exportDataRequest.
Same will be done for `allColumnsOfType` in another PR.

**Review by yecli — APPROVED** [review]

File: Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-type-selector.js (hunk 2 above)
Comment (akak1977) [reply Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-type-selector.js]: I'm already done it for allColumnsOfType

Comment (yecli) [reply Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-type-selector.js]: @akak1977 Please take this changes into your branch, as it is already merged.

**Pull request merged**